### ISSUE TITLE
[Visual Requirement-Header Control-Projects]: Luminosity ratio is less than 4.5:1 for some Text showing under Project dialog box.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2519]
**User Experience**
If the luminosity ratio is less than 4.5:1 for all the text content then it will be difficult for the visually impaired user to see what is present on the screen.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.
Tool used:-Color contrast Analyzer

**Repro Steps**
1. Navigate to https://makecode.microbit.org/acc
2. Navigate to the Microbit section element and select Code control given on it.
3. Navigate to the Projects control lying in the header section on the page and select it.
4. Navigate to various controls lying on the page opened.
5. Verify the luminosity ratio for all the Text content.
 
**Actual Result**
Luminosity ratio is less than 4.5:1 for some Text(Project tab, Examples Tab,Today,8minutes ago) showing under Project dialog box .

**Expected Result**
Luminosity ratio should be more than 4.5:1 for some Text(Project tab, Examples Tab,Today,8minutes ago) showing under Project dialog box .

**MAS Reference** 
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Suggested fix**
1.1-Ensuring that a contrast ratio of at least 4.5:1 exists between text (and images of text) and background behind the text 
1.2- If design allows , Not specify background color, not specify text color, and not using technology features that change those defaults .In this technique the author avoids having to do any contrast measures by simply not specifying the text color and not specifying the background. As a result the colors of both are completely determined by the user agent.
Tips:-
•The easiest fix is to increase the brightness of light colors and decrease the brightness of darker colors. 
•Exaggerate lightness differences between foreground and background colors, and avoid using colors of similar brightness adjacent to one another, even if they differ in saturation or hue. 
•During design Use the Color Contrast Analyzer to determine whether new colors have color contrast ratio of at least 4.5:1. 
 
**Key References for Fix**
http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html

**Please refer the attachment**
![mas1 4 3_header controls_projects](https://user-images.githubusercontent.com/30459830/28609143-06af9ae4-7201-11e7-90f8-15a0a40d0a3b.jpg)

**Comment by v-amik** [comment]
review done

**Comment by v-mesala** [comment]
Bug is verified on URL : https://makecode.microbit.org/acc/ and bug still exists as Luminosity ratio is still less than 4.5:1 for the Time showing under Today Section.Hence recativated the bug.

**Comment by veler** [comment]
I realized that this issue is part of an opened PR in PXT-microbit ([https://github.com/Microsoft/pxt-microbit/pull/466](https://github.com/Microsoft/pxt-microbit/pull/466)) and need to be discussed as it's part of the Micro:bit color theme.

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/acc and issue is fixed now. Hence closing the bug
